lms/djangoapps/ccx/views.py

```diff
@@ -572,6 +576,28 @@ def ccx_course(ccx_locator):
     yield course
 
 
+def add_master_course_staff_to_ccx(course, ccx_key, display_name):
+    """
+    Assign staff of master course, staff role on ccx.
+    :param course: Master course instance
+    :param ccx_key: CCX course key
+    :param display_name: ccx display name for email
+    """
+    list_staff = list_with_level(course, 'staff')
+    with ccx_course(ccx_key) as course:
+        email_params = get_email_params(course, auto_enroll=True, course_key=ccx_key, display_name=display_name)
+        for staff in list_staff:
+            allow_access(course, staff, 'staff')
+            # Enroll the coach in the course
+            enroll_email(
+                course_id=ccx_key,
+                student_email=staff.email,
+                auto_enroll=True,
+                email_students=True,
+                email_params=email_params,
+            )
+
+
```

Comment:
I think this function looks fine, but what I meant in my comment in the story was to put this function in an external module ( `api.py` ?) so I can import it easily.